channels-rke2.yaml

```diff
@@ -2168,7 +2168,7 @@ releases:
     featureVersions: *featureVersions-v1
   - version: v1.28.14+rke2r1
     minChannelServerVersion: v2.8.3-alpha1
-    maxChannelServerVersion: v2.10.99
+    maxChannelServerVersion: v2.9.99
```

Comment:
Why are we changing these to `v2.9.99`? iirc the `maxChannelServerVersion` for this branch (`dev-v2.10`) should all be `v2.10.99`.

Also, there're some changes that were made without the prefix `v`.